### PR TITLE
translate 2014/09/19/ruby-2-0-0-p576-is-released into zh_cn

### PR DESCRIPTION
**Pull request by li-thy-um**
Related #1074.
ruby-2-0-0-p576-is-released

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@pmq20 @hlcfan @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi @shouxian @johnnyhappy365
@aidewoode @Yunfei1982 @Airmacho @cursorzz @isjia @debbbbie

Please review.

File: zh_cn/news/_posts/2014-09-19-ruby-2-0-0-p576-is-released.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+layout: news_post
+title: "Ruby 2.0.0-p576 今日发布"
+author: "usa"
+translator: "li-thy-um"
+date: 2014-09-19 12:00:00 +0000
+lang: zh_cn
+---
+
+我们很高兴地宣布发布 Ruby 2.0.0-p576, 以庆祝正在日本召开的 [RubyKaigi2014](http://rubykaigi.org/2014) 。
```
Comment (1ternal) [inline zh_cn/news/_posts/2014-09-19-ruby-2-0-0-p576-is-released.md]: comma should be  "，" :lollipop:

File: zh_cn/news/_posts/2014-09-19-ruby-2-0-0-p576-is-released.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+layout: news_post
+title: "Ruby 2.0.0-p576 今日发布"
+author: "usa"
+translator: "li-thy-um"
+date: 2014-09-19 12:00:00 +0000
+lang: zh_cn
+---
+
+我们很高兴地宣布发布 Ruby 2.0.0-p576， 以庆祝正在日本召开的 [RubyKaigi2014](http://rubykaigi.org/2014) 。
```
Comment (hlcfan) [inline zh_cn/news/_posts/2014-09-19-ruby-2-0-0-p576-is-released.md]: Hmm, seems an extra space here. **Ruby 2.0.0-p576， 以庆祝**

**Comment by 1ternal** [comment]
@JuanitoFatas  I think it's good to go  :100:

**Comment by JuanitoFatas** [comment]
Thanks guys. Merged manually as 611fc40.

**Pull request closed without merging**